lib/decision_review_v1/utilities/helpers.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
 require 'decision_review_v1/utilities/constants'
+require 'decision_review_v1/utilities/logging_utils'
 
 module DecisionReviewV1
   module Appeals
     module Helpers
+      include DecisionReviewV1::Appeals::LoggingUtils
```

Comment:
Including this for backwards compatibility, though we may consider keeping these modules completely separate in the future.